.github/workflows/blackduck_scan.yml

```diff
@@ -14,6 +14,17 @@ on:
           - v6
           - v7
         default: v7
+      log-level:
+        description: 'Log level of scanning. Use DEBUG or TRACE for troubleshooting.'
+        type: choice
+        options:
+          - 'OFF'
```

Comment:
Why only 'OFF' has the single quote?

Reply:
Per YML spec, as far as I can remember version 1.1, `OFF` is an alias for boolean `false`.

Reply:
https://yaml.org/type/bool.html